site/content/en/docs/Guides/metrics.md

```diff
@@ -54,10 +55,11 @@ Finally include that `ServiceMonitor` in your [Prometheus instance CRD](https://
 
 ### Stackdriver
 
-We don't yet support the [OpenCensus Stackdriver exporter](https://opencensus.io/exporters/supported-exporters/go/stackdriver/)
-but you can still use the Prometheus Stackdriver integration by following these [instructions](https://cloud.google.com/monitoring/kubernetes-engine/prometheus).
-Annotations required by this integration can be activated by setting the `agones.metrics.prometheusServiceDiscovery` 
-to true (default) via the [helm chart value]({{< relref "../Installation/helm.md" >}}).
+We support the [OpenCensus Stackdriver exporter](https://opencensus.io/exporters/supported-exporters/go/stackdriver/). 
```

Comment:
You get to play with our new documentation tools! We have "shortcodes" which you can wrap content in, to hide and show previous versions. See: https://agones.dev/site/docs/contribute/ for details.

TL;DR -

The old version should we wrapped in:

```
{{% feature expiryVersion="0.8.0" %}}
We don't yet support the OpenCensus Stackdriver exporter...
{{\% /feature %}}
```

And the new documentation should be wrapped in:
```
{{% feature publishVersion="0.8.0" %}}
We support the OpenCensus Stackdriver exporter...
{{\% /feature %}}
```

The tooling will show/hide each section appropriately as we release new versions.